packages/core/src/node/logger/hint.ts

```diff
@@ -44,6 +45,35 @@ export async function hintThemeBreakingChange(customThemeDir: string) {
   }
 }
 
+export function hintBuilderPluginsBreakingChange(config: UserConfig) {
+  if (
+    // @ts-ignore config.builderPlugins is removed in V2
+    config.builderPlugins &&
```

Comment:
```suggestion
    'builderPlugins' in config &&
```

I think this coule avoid `//@ts-ignore` as much as possible.

By the way, I'd always prefer `//ts-expect-error` wherever possible.